open-data/Lesson6-Sharing&Reuse.md

```diff
@@ -0,0 +1,311 @@
+* Wikidata[ https://www.wikidata.org/wiki/Wikidata:Main_Page](https://www.wikidata.org/wiki/Wikidata:Main_Page)
+* Open Data Network [https://www.opendatanetwork.com/](https://www.opendatanetwork.com/)
+* Awesome Public Datasets[ https://github.com/awesomedata/awesome-public-datasets#readme](https://github.com/awesomedata/awesome-public-datasets#readme)
+
+### **Examples of Discipline specific:**
+
+* NASA Earth[ https://www.earthdata.nasa.gov/](https://www.earthdata.nasa.gov/)
+* Cern[ https://opendata.cern.ch/](https://opendata.cern.ch/)
+* NCBI National Center for Biotechnology Information[ https://www.ncbi.nlm.nih.gov/](https://www.ncbi.nlm.nih.gov/)
+* EMBL's European Bioinformatics Institute[ https://www.ebi.ac.uk/](https://www.ebi.ac.uk/)
+* ISPCR[ https://www.icpsr.umich.edu/web/pages/](https://www.icpsr.umich.edu/web/pages/)
+* International Monetary Fund  [https://www.imf.org/en/Data](https://www.imf.org/en/Data)
+* NOAA Climate Data Online [https://www.ncdc.noaa.gov/cdo-web/datasets](https://www.ncdc.noaa.gov/cdo-web/datasets)  
+* Federal Reserve Economic Research [https://fred.stlouisfed.org/](https://fred.stlouisfed.org/)
+* USGS EarthExplorer [https://earthexplorer.usgs.gov/](https://earthexplorer.usgs.gov/)
+* Open Science Data Cloud (OSDC) [https://www.opensciencedatacloud.org/](https://www.opensciencedatacloud.org/)
+* NASA Planetary Data System [https://pds.nasa.gov/](https://pds.nasa.gov/)
+
+
+### **Examples of National or international data portal**
+
+* US Federal data[ https://data.gov/](https://data.gov/)
+* EU Data Portal[ https://data.europa.eu/en](https://data.europa.eu/en)
+* WHO[ https://apps.who.int/gho/data/node.home](https://apps.who.int/gho/data/node.home)
+* THE WORLD BANK [https://data.worldbank.org/](https://data.worldbank.org/)
+* DATA.GOV.UK [https://www.data.gov.uk/](https://www.data.gov.uk/)
+* UNICEF [https://data.unicef.org/](https://data.unicef.org/)
+
+
+### Literature search
+
+While not ideal, datasets are often attached to scholarly publications in the form of supplementary material, or referenced in text where to find them e.g. GitHub repository or personal/institutional websites. In addition, there are emerging journals and special collections/issues focused on describing and publishing data (e.g. Nucleic Acids Research database issues[ https://doi.org/10.1093/nar/gkab1195](https://doi.org/10.1093/nar/gkab1195), Scientific Data, Earth System Science Data, etc.). In other words, while the datasets are openly available in these media, they are not properly indexed and therefore not very findable nor machine readable.
+
+Finding academic publications can be a challenge in itself depending on the discipline and field of study. For instance, in life science and biomedical research, there are a number of repositories and search engines (e.g. PubMed, EuropePMC) indexing research outputs (e.g. publications, abstracts, references and communications) from various journals.
```

Comment:
```suggestion
Finding academic publications can be a challenge in itself depending on the discipline and field of study. For life and   biomedical research it is less challenging, since there are a number of repositories and search engines (PubMed, EuropePMC) indexing research outputs from various journals. 
```